en/docs/trust/wso2-public-cloud-data-protection-faq.md

```diff
@@ -0,0 +1,441 @@
+
+**As a Data processor, WSO2 will be responsible for:**
+* Processing Personal Data in accordance with the Data Controller´s instructions.
+* Safeguarding and protecting the Data Controller's data in accordance with all required technical and organizational measures and data protection laws currently in force.
+* Ensuring Data subject requests (DSRs) are addressed in timely manner.
+* Providing information and all reasonable assistance related to data privacy and security requests from the Data controller.
+* Ensuring that sub-processors adhere to all data protection requirements and standards.
+* Ensuring that WSO2 has adopted appropriate safeguards and adequate levels of protection for cross-border data transfers.
+* Notifying the Data controller if WSO2:
+  * receive requests from the subscriber's administrators and developers exercising their GDPR rights.
+  * receive requests from any supervisory authorities (unless prohibited by law).
+  * receive requests from any law enforcement authorities.
+  * becomes aware of a confirmed security breach.
+  * changes to privacy policies, terms, security statements, data protection agreements, processors, or sub-processors.
+
+### Subscriber's Role in Protecting Data
+
+**The subscriber, as the Data controller, will be responsible for:**
+* Complying with relevant data protection laws and regulations as applicable to the subscriber.
+* As between WSO2 and the subscriber, the subscriber will always remain in control of the data added by the subscriber or its users to the WSO2 Public Clouds.
+* Ensuring that any applications, websites, APIs, components, and integrations subscribers develop, deploy, or host on WSO2 Public Clouds are securely designed, developed, and managed in accordance with the subscriber's organizational security policies and best practices.
+* Adhering to relevant security best practices as per WSO2 product documentation or of the relevant information technology being used.
+* Inform WSO2 of any data subject requests from subscribers' administrators and developers so that WSO2 can support the subscriber in processing the request. The subscribers can submit a request [form](https://wso2.com/data-privacy-protection-request/) or contact the data protection officer at dpo@wso2.com.
+* Informing WSO2 of any vulnerabilities or security issues related to the platform using support channels or [Security Reporting Channels](https://security.docs.wso2.com/en/latest/security-reporting/report-security-issues/).
+* Complying with applicable WSO2 Public Cloud Terms of Use.
+
+## Data Residency
+
+### Where is the subscriber data stored?
+* **Asgardeo**
+  * Asgardeo organization administrator profiles and associated data would be stored in Asgardeo US deployment.
```

Comment:
Asgardeo organization administrator profiles and associated data are stored in the region where the deployment is provisioned (US or EU) after the 1.5 migration